__init__.py

```diff
@@ -558,35 +633,57 @@ def play_playlist_on(self, message):
                     self.spotify.transfer_playback(dev["id"], False)
                 self.start_playback(dev, playlist)
 
-    @intent_file_handler('PlaySpotify.intent')
-    def continue_current_playlist(self, message):
-        if self.playback_prerequisits_ok():
-            dev = self.get_default_device()
-            if dev:
-                self.spotify_play(dev['id'])
-            else:
-                self.speak_dialog('NoDevicesAvailable')
+#    @intent_file_handler('PlaySpotify.intent')
+#    def continue_current_playlist(self, message):
+#        if self.playback_prerequisits_ok():
+#            dev = self.get_default_device()
+#            if dev:
+#                self.spotify_play(dev['id'])
+#            else:
+#                self.speak_dialog('NoDevicesAvailable')
 
-    @intent_handler(IntentBuilder('').require('Search').
-                    require('AlbumTitle').require('Spotify'))
-    def search_album(self, message):
-        if self.playback_prerequisits_ok():
-            return self.search(message.data['AlbumTitle'], 'album')
-
-    def play_album(self, message):
-        if self.playback_prerequisits_ok():
-            return self.search(message.data['AlbumTitle'], 'album')
+    def play(self, data, type='track', genreName=None):
+        """
 
-    def play_artist(self, message):
-        if self.playback_prerequisits_ok():
-            return self.search(message.data['Artist'], 'artist')
+        :param data: data returned by self.search_spotify
+        :param type: the type of data. 'track', 'album', or 'genre' are currently supported
+        :param genreName: if type is 'genre', also include the genre's name here, for output purposes
```

Comment:
We try to use the google style docstrings: http://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html

Reply:
I've rewritten them to be in that style